# Represents the RPM value of the left and right motors 0...1
float32 rpm_left
float32 rpm_right